src/boot/index.js

```diff
@@ -21,6 +21,18 @@ if (isBrowserSupported()) {
   const config = /** @type {AnnotatorConfig|SidebarAppConfig} */ (
     parseJsonConfig(document)
   );
+
+  // When the boot script is executed from the browser extension, at least one
+  // config is required
+  const isExtensionContext = !!(
+    /** @type {any} */ (window).chrome?.runtime?.id
+  );
+  if (!Object.keys(config).length && isExtensionContext) {
```

Comment:
It is possible the page may contain its own Hypothesis configuration. What we really want to check for here is the config that corresponds to the specific extension being used. One way to do this might be to add an attribute to the configuration `<script>` indicating the ID of the extension

Reply:
Yeah, you are right. In fact I run into this while testing on a page that precisely had its own config 😅 

I like the idea of the attribute with the extension ID. I'm going to merge this as is, address your suggestion in the browser extension, and then come back here and improve it.

Thanks!.